bin/crew

```diff
@@ -129,7 +142,7 @@ def load_json
 end
 
 def save_json(json_object)
-  crewlog 'Saving device.json...'
+  # crewlog 'Saving device.json...'
```

Comment:
Uncomment?  If verbose is not set, this won't print anyway, right?

Reply:
I was just try to clean up stuff I wasn't using to debug any more... I should just delete these...

Reply:
I'll just uncomment them...